src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const Todo = [
```

Comment:
It may help to put this array in the state of the App component. This way you can pass it down as props to the TodoList component where you can iterate the array and display each Todo.